dockerfiles/compilation/java-8/compile.sh

```diff
@@ -44,17 +45,27 @@ git checkout $SHA1
 # Compilation
 
 if [ "$SKIPTESTS" = "skipTests" ]; then
-   mvn package -DskipTests -Pcassandra,elasticsearch,inmemory,with-assembly,with-jetm
+   mvn package -DskipTests -Pcassandra,inmemory,jpa,elasticsearch,lucene,with-assembly,with-jetm
 else
-   mvn package -Pcassandra,inmemory,elasticsearch,with-assembly,with-jetm
+   mvn package -Pcassandra,inmemory,jpa,elasticsearch,lucene,with-assembly,with-jetm
 fi
 
 # Retrieve result
 
 if [ $? -eq 0 ]; then
-   cp server/app/target/james-server-app-*-app.zip $DESTINATION
-   cp server/container/guice/cassandra-guice/target/james-server-cassandra-guice-*-SNAPSHOT.jar $DESTINATION
-   cp -r server/container/guice/cassandra-guice/target/james-server-cassandra-guice-*-SNAPSHOT.lib $DESTINATION
-   cp server/container/cli/target/james-server-cli-*.jar $DESTINATION
-   cp -r server/container/cli/target/james-server-cli-*.lib $DESTINATION
+   if [ -d "$CASSANDRA_DESTINATION" ]; then
```

Comment:
while we are at it, we could remove the version from artifact names so we don't have to change all dockerfile when we change version string.